plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8453,6 +8453,31 @@ public void testCreateAcidTableUnsupported()
         assertQueryFails("CREATE TABLE acid_unsupported WITH (transactional = true) AS SELECT 123 x", "FileHiveMetastore does not support ACID tables");
     }
 
+    @Test
```

Comment:
Can we add additional test with altering table properties ? And what if we try to override one of the existing extra table properties ?

Reply:
Do you mean a test with altering the table properties through hive? I can't write a test doing this with the hive connector in Trino since it doesn't support `ALTER TABLE <xyz> SET PROPERTIES`.